controllers/controller.go

```diff
@@ -0,0 +1,78 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/gror/model"
+	"github.com/gror/services"
+	"gopkg.in/mgo.v2/bson"
+)
+
+func CreateDocker(w http.ResponseWriter, r *http.Request) {
```

Comment:
Update the method name to something like `CreateDockerConfig` and also make the same changes for other methods